src/renderer/extensions/vueNodes/widgets/components/WidgetTreeSelect.test.ts

```diff
@@ -0,0 +1,547 @@
+import { mount } from '@vue/test-utils'
+import PrimeVue from 'primevue/config'
+import TreeSelect from 'primevue/treeselect'
+import type { TreeSelectProps } from 'primevue/treeselect'
+import { describe, expect, it, vi } from 'vitest'
+
+import type { SimplifiedWidget, WidgetValue } from '@/types/simplifiedWidget'
+
+import WidgetTreeSelect from './WidgetTreeSelect.vue'
+
+type TreeNode = {
+  key: string
+  label?: string
+  data?: unknown
+  children?: TreeNode[]
+  leaf?: boolean
+  selectable?: boolean
+}
```

Comment:
Same question as in the imagecompare PR, is this type available from within the component code?